Add separate pipeline for CI and fix cleanup

ecraig12345 opened the pull request:
### Partially separating PR/CI pipelines

Change our main pipeline file to only include a trigger for PRs, and have CI use a [separate pipeline](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=164&_a=summary) which uses the same file but with a [trigger override in the UI](https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers). This has multiple benefits:
- Makes it easier to get meaningful [analytics data](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=84&view=ms.vss-pipelineanalytics-web.new-build-definition-pipeline-analytics-view-cardmetrics) about build failure rates
- Fixes Teams notifications for failed builds: we can use the Azure Pipelines bot to subscribe to only the CI pipeline (since the bot doesn't have branch filters, for the combined pipeline I had it set it up to email the channel if master or 7.0 build failed...but then the email got too bulky for Teams to display)
- Allows us to restrict what resources the PR pipeline can access (future, not set up today)

Along with this, I added better conditional logic to reduce duplication of pipeline steps between PR and CI.

### Fixing component governance detection

I also noticed that my previous attempt in #17147 to address the "Directory not empty" error caused the automatic component governance detection task to fail (this is injected by a pipeline decorator from an extension, and only runs against master). This causes confusing [warnings](https://dev.azure.com/uifabric/fabricpublic/_build/results?buildId=191796&view=results) and means component governance detection doesn't work. 

As a workaround, I started setting a variable `skipComponentGovernanceDetection: true` in the shared variables template by default (per their docs this should prevent the decorator from running). Then I added a manual run of the component governance detection task in the release build only. Running the task in only one pipeline should be fine since the point of the task is to check installed packages, and we install the same packages in all our pipelines in each branch.

ling1726 commented on .devops/templates/cleanup.yml:
```diff
@@ -2,13 +2,36 @@ parameters:
+  # (For example, CI builds only need to run this on one of the jobs, not every job.)
+  # prettier-ignore
+  - ${{ if and(eq(parameters.runComponentGovernance, true), eq(variables['Build.SourceBranch'], 'refs/heads/master')) }}:
+    - script: echo "##vso[task.setvariable variable=skipComponentGovernanceDetection;]true"
```
if we can disable, why not just setup another job that runs component governance ?

ecraig12345 replied:
...yeah that would probably make sense 😆

ecraig12345 replied:
Dang ~~boring~~ simple ways to do things, now nobody gets to see my fancy yaml syntax :P

ecraig12345 replied:
Updated the tools/setup template to disable the job by default, and added a step to `azure-pipelines.release.yml` to run component governance detection manually. As far as I can tell this should be adequate since the point of the task is to check installed packages, and we install the same packages in all our pipelines in each branch.

ling1726 commented on .devops/templates/pr-ci.yml:
```diff
@@ -0,0 +1,159 @@
+          yarn prettier $(targetBranch) --check
+        displayName: do syncpack, lint-files, and prettier checks
+
+      ## Danger.js checks for Fluent UI N*
```
with the recent work on separating v0 from v7/8 would it make sense to just spin up a separate pipeline for v0 ? 

We removed dependency on datetime and keyboard key, I don't think there's much left that really ties it with the rest of the repo

ecraig12345 replied:
Maybe, but the trouble is figuring out how to do the scoping. Also it's a bit of an orthogonal problem to what I'm trying to solve here.

ling1726 replied:
I agree, just wanted to float the idea, it will also make both pipelines (if we get there) a lot lot simpler

ling1726 commented on azure-pipelines.ci.yml:
```diff
@@ -1 +1,9 @@
+extends:
+  template: .devops/templates/pr-ci.yml
+  parameters:
+    isPR: true
```
the `isPR` value is `true` for both PR and master pipelines is that intentional ?

ecraig12345 replied:
no 🤦‍♀️  fixed now

fabricteam commented:
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Avatar-mount_base.html">1226</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Avatar-mount.html">1231</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/BaseButton-mount_base.html">992</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/BaseButton-mount.html">999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Breadcrumb-mount_base.html">44555</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Breadcrumb-mount.html">44058</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ButtonNext-mount_base.html">1438</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ButtonNext-mount.html">1423</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Checkbox-mount_base.html">1653</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Checkbox-mount.html">1661</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/CheckboxBase-mount_base.html">1388</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/CheckboxBase-mount.html">1434</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ChoiceGroup-mount_base.html">5224</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ChoiceGroup-mount.html">5208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ComboBox-mount_base.html">1067</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ComboBox-mount.html">992</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/CommandBar-mount_base.html">10487</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/CommandBar-mount.html">10632</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ContextualMenu-mount_base.html">6378</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ContextualMenu-mount.html">6412</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DefaultButton-mount_base.html">1232</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DefaultButton-mount.html">1231</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRow-mount_base.html">3782</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRow-mount.html">3837</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRowFast-mount_base.html">4099</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRowFast-mount.html">3874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRowNoStyles-mount_base.html">3716</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DetailsRowNoStyles-mount.html">3661</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Dialog-mount_base.html">1610</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Dialog-mount.html">1583</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DocumentCardTitle-mount_base.html">1910</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/DocumentCardTitle-mount.html">1872</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Dropdown-mount_base.html">3608</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Dropdown-mount.html">3589</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/FocusTrapZone-mount_base.html">1900</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/FocusTrapZone-mount.html">1897</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/FocusZone-mount_base.html">1791</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/FocusZone-mount.html">1872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/IconButton-mount_base.html">1923</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/IconButton-mount.html">1931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Label-mount_base.html">337</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Label-mount.html">343</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Layer-mount_base.html">1876</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Layer-mount.html">1860</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Link-mount_base.html">471</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Link-mount.html">476</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MakeStyles-mount_base.html">2130</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MakeStyles-mount.html">2076</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MenuButton-mount_base.html">1594</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MenuButton-mount.html">1575</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MessageBar-mount_base.html">2085</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/MessageBar-mount.html">2108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Nav-mount_base.html">3544</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Nav-mount.html">3616</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/OverflowSet-mount_base.html">1064</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/OverflowSet-mount.html">1112</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Panel-mount_base.html">1510</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Panel-mount.html">1426</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Persona-mount_base.html">846</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Persona-mount.html">873</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Pivot-mount_base.html">1490</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Pivot-mount.html">1469</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/PrimaryButton-mount_base.html">1372</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/PrimaryButton-mount.html">1399</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Rating-mount_base.html">8315</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Rating-mount.html">8213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SearchBox-mount_base.html">1422</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SearchBox-mount.html">1419</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Shimmer-mount_base.html">2730</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Shimmer-mount.html">2732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Slider-mount_base.html">2113</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Slider-mount.html">2117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SpinButton-mount_base.html">5339</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SpinButton-mount.html">5410</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Spinner-mount_base.html">437</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Spinner-mount.html">440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SplitButton-mount_base.html">3368</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SplitButton-mount.html">3390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Stack-mount_base.html">532</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Stack-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/StackWithIntrinsicChildren-mount_base.html">1688</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/StackWithIntrinsicChildren-mount.html">1663</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/StackWithTextChildren-mount_base.html">5029</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/StackWithTextChildren-mount.html">4964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SwatchColorPicker-mount_base.html">10896</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/SwatchColorPicker-mount.html">10921</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Tabs</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Tabs-mount_base.html">1489</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Tabs-mount.html">1485</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TagPicker-mount_base.html">3000</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TagPicker-mount.html">2989</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TeachingBubble-mount_base.html">12197</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TeachingBubble-mount.html">12228</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Text-mount_base.html">458</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Text-mount.html">443</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TextField-mount_base.html">1439</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/TextField-mount.html">1534</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProvider-mount_base.html">1277</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProvider-mount.html">1248</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProvider-virtual-rerender_base.html">633</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProvider-virtual-rerender.html">629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProviderNext</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProviderNext-mount_base.html">16261</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/ThemeProviderNext-mount.html">16133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Toggle-mount_base.html">848</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/Toggle-mount.html">902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/buttonNative-mount_base.html">128</a></td>
            <td><a href="http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test/buttonNative-mount.html">127</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (Fluent)
<details><summary>Perf comparison</summary>

Status | Scenario | Fluent TPI | Fabric TPI | Ratio | Iterations | Ticks
:---: | :--- | ---:| ---:| ---:| ---:| ---:
🦄 | Avatar.Fluent | [0.2](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Avatar.Fluent.html) | 0.52 | 0.38:1 | 2000 | 409
🦄 | Button.Fluent | [0.14](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Button.Fluent.html) | 0.23 | 0.61:1 | 5000 | 680
🔧 | Checkbox.Fluent | [0.71](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Checkbox.Fluent.html) | 0.38 | 1.87:1 | 1000 | 712
🎯 | Dialog.Fluent | [0.17](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dialog.Fluent.html) | 0.24 | 0.71:1 | 5000 | 867
🔧 | Dropdown.Fluent | [3.3](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dropdown.Fluent.html) | 0.47 | 7.02:1 | 1000 | 3295
🔧 | Icon.Fluent | [0.15](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Icon.Fluent.html) | 0.07 | 2.14:1 | 5000 | 738
🦄 | Image.Fluent | [0.09](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Image.Fluent.html) | 0.14 | 0.64:1 | 5000 | 458
🔧 | Slider.Fluent | [1.67](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Slider.Fluent.html) | 0.49 | 3.41:1 | 1000 | 1671
🔧 | Text.Fluent | [0.09](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Text.Fluent.html) | 0.04 | 2.25:1 | 5000 | 443
🦄 | Tooltip.Fluent | [0.13](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Tooltip.Fluent.html) | 0.95 | 0.14:1 | 5000 | 635
>🔧 Needs work &nbsp; &nbsp; 🎯 On target &nbsp; &nbsp; 🦄 Amazing

</details>
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
IconMinimalPerf.default | [858](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/IconMinimalPerf.default.html) | [743](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.15:1
PortalMinimalPerf.default | [201](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/PortalMinimalPerf.default.html) | [177](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.14:1
RadioGroupMinimalPerf.default | [573](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [513](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.12:1
ButtonMinimalPerf.default | [225](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonMinimalPerf.default.html) | [207](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.09:1
RefMinimalPerf.default | [278](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RefMinimalPerf.default.html) | [257](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.08:1
GridMinimalPerf.default | [437](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/GridMinimalPerf.default.html) | [410](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.07:1
ImageMinimalPerf.default | [469](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ImageMinimalPerf.default.html) | [439](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.07:1
AnimationMinimalPerf.default | [479](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AnimationMinimalPerf.default.html) | [455](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.05:1
AttachmentSlotsPerf.default | [1376](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1316](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.05:1
StatusMinimalPerf.default | [852](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/StatusMinimalPerf.default.html) | [815](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.05:1
AlertMinimalPerf.default | [357](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AlertMinimalPerf.default.html) | [344](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.04:1
ChatMinimalPerf.default | [729](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatMinimalPerf.default.html) | [702](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.04:1
FormMinimalPerf.default | [524](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/FormMinimalPerf.default.html) | [506](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.04:1
ListCommonPerf.default | [753](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListCommonPerf.default.html) | [727](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListCommonPerf.default_base.html) | 1.04:1
LoaderMinimalPerf.default | [810](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LoaderMinimalPerf.default.html) | [778](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.04:1
Text.Fluent | [443](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Text.Fluent.html) | [424](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Text.Fluent_base.html) | 1.04:1
AttachmentMinimalPerf.default | [207](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AttachmentMinimalPerf.default.html) | [201](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.03:1
FlexMinimalPerf.default | [361](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/FlexMinimalPerf.default.html) | [350](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.03:1
MenuButtonMinimalPerf.default | [1843](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1781](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.03:1
SkeletonMinimalPerf.default | [445](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SkeletonMinimalPerf.default.html) | [433](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.03:1
Avatar.Fluent | [409](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Avatar.Fluent.html) | [399](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Avatar.Fluent_base.html) | 1.03:1
Checkbox.Fluent | [712](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Checkbox.Fluent.html) | [692](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Checkbox.Fluent_base.html) | 1.03:1
ButtonSlotsPerf.default | [686](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonSlotsPerf.default.html) | [671](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.02:1
HeaderSlotsPerf.default | [920](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/HeaderSlotsPerf.default.html) | [899](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.02:1
InputMinimalPerf.default | [1433](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/InputMinimalPerf.default.html) | [1403](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.02:1
ListNestedPerf.default | [680](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListNestedPerf.default.html) | [664](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListNestedPerf.default_base.html) | 1.02:1
ListWith60ListItems.default | [704](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListWith60ListItems.default.html) | [688](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.02:1
PopupMinimalPerf.default | [808](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/PopupMinimalPerf.default.html) | [792](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
ProviderMinimalPerf.default | [1109](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ProviderMinimalPerf.default.html) | [1088](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.02:1
ReactionMinimalPerf.default | [481](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ReactionMinimalPerf.default.html) | [471](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.02:1
SplitButtonMinimalPerf.default | [4213](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4147](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.02:1
ToolbarMinimalPerf.default | [1113](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ToolbarMinimalPerf.default.html) | [1095](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.02:1
TooltipMinimalPerf.default | [914](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TooltipMinimalPerf.default.html) | [897](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.02:1
Button.Fluent | [680](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Button.Fluent.html) | [669](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Button.Fluent_base.html) | 1.02:1
Dropdown.Fluent | [3295](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dropdown.Fluent.html) | [3238](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dropdown.Fluent_base.html) | 1.02:1
ButtonUseCssPerf.default | [935](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonUseCssPerf.default.html) | [925](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonUseCssPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [834](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DropdownManyItemsPerf.default.html) | [826](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
LayoutMinimalPerf.default | [475](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LayoutMinimalPerf.default.html) | [469](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.01:1
ListMinimalPerf.default | [595](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListMinimalPerf.default.html) | [589](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [2394](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TableManyItemsPerf.default.html) | [2379](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
Dialog.Fluent | [867](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dialog.Fluent.html) | [858](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Dialog.Fluent_base.html) | 1.01:1
Tooltip.Fluent | [635](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Tooltip.Fluent.html) | [629](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Tooltip.Fluent_base.html) | 1.01:1
BoxMinimalPerf.default | [445](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/BoxMinimalPerf.default.html) | [445](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1:1
ButtonOverridesMissPerf.default | [1868](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1862](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1:1
DividerMinimalPerf.default | [431](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DividerMinimalPerf.default.html) | [431](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [4696](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/EmbedMinimalPerf.default.html) | [4690](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
LabelMinimalPerf.default | [492](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LabelMinimalPerf.default.html) | [494](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1:1
SliderMinimalPerf.default | [1700](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SliderMinimalPerf.default.html) | [1694](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1:1
TableMinimalPerf.default | [477](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TableMinimalPerf.default.html) | [475](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TableMinimalPerf.default_base.html) | 1:1
TextMinimalPerf.default | [437](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TextMinimalPerf.default.html) | [438](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TextMinimalPerf.default_base.html) | 1:1
TextAreaMinimalPerf.default | [589](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TextAreaMinimalPerf.default.html) | [590](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [4047](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CustomToolbarPrototype.default.html) | [4030](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
ButtonUseCssNestingPerf.default | [1208](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonUseCssNestingPerf.default.html) | [1216](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ButtonUseCssNestingPerf.default_base.html) | 0.99:1
CarouselMinimalPerf.default | [539](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CarouselMinimalPerf.default.html) | [542](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.99:1
CheckboxMinimalPerf.default | [3077](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CheckboxMinimalPerf.default.html) | [3107](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 0.99:1
DatepickerMinimalPerf.default | [51771](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DatepickerMinimalPerf.default.html) | [52152](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 0.99:1
DialogMinimalPerf.default | [892](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DialogMinimalPerf.default.html) | [902](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DialogMinimalPerf.default_base.html) | 0.99:1
HeaderMinimalPerf.default | [448](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/HeaderMinimalPerf.default.html) | [451](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 0.99:1
ItemLayoutMinimalPerf.default | [1379](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1389](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 0.99:1
TreeMinimalPerf.default | [893](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TreeMinimalPerf.default.html) | [899](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TreeMinimalPerf.default_base.html) | 0.99:1
Slider.Fluent | [1671](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Slider.Fluent.html) | [1680](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Slider.Fluent_base.html) | 0.99:1
ChatDuplicateMessagesPerf.default | [406](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [415](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.98:1
ChatWithPopoverPerf.default | [491](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatWithPopoverPerf.default.html) | [502](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.98:1
DropdownMinimalPerf.default | [3209](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DropdownMinimalPerf.default.html) | [3273](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 0.98:1
MenuMinimalPerf.default | [1023](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/MenuMinimalPerf.default.html) | [1049](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.98:1
ProviderMergeThemesPerf.default | [1722](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1759](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 0.98:1
TreeWith60ListItems.default | [196](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TreeWith60ListItems.default.html) | [200](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/TreeWith60ListItems.default_base.html) | 0.98:1
Image.Fluent | [458](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Image.Fluent.html) | [467](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Image.Fluent_base.html) | 0.98:1
AccordionMinimalPerf.default | [185](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AccordionMinimalPerf.default.html) | [191](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.97:1
CardMinimalPerf.default | [659](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CardMinimalPerf.default.html) | [680](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.97:1
VideoMinimalPerf.default | [727](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/VideoMinimalPerf.default.html) | [746](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/VideoMinimalPerf.default_base.html) | 0.97:1
AvatarMinimalPerf.default | [234](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AvatarMinimalPerf.default.html) | [245](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 0.96:1
RosterPerf.default | [1281](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RosterPerf.default.html) | [1341](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/RosterPerf.default_base.html) | 0.96:1
SegmentMinimalPerf.default | [427](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SegmentMinimalPerf.default.html) | [449](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.95:1
Icon.Fluent | [738](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Icon.Fluent.html) | [801](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17184/merge/perf-test-northstar/Icon.Fluent_base.html) | 0.92:1

</details><div id="perfComment9423"/>

Hotell commented:
I guess this is good step forward. sidenote: what we should rather invest  heavily is context aware pipeline - there is excellent oss tooling for that.

> automatic component governance detection

what is that ? can you point me to docs ?

layershifter commented on .devops/templates/pr-ci.yml:
```diff
@@ -0,0 +1,157 @@
+
+      - template: cleanup.yml
+
+  - job: ScreenerFluent
```
```suggestion
  - job: ScreenerNorthstar
```

nit: As we touched this file, can we improve naming of this job? 🐱

ecraig12345 replied:
done

ecraig12345 commented:
> > automatic component governance detection
> 
> what is that ? can you point me to docs ?

@Hotell It's an internal thing, I'll send you links on Teams

ecraig12345 commented:
> sidenote: what we should rather invest heavily is context aware pipeline - there is excellent oss tooling for that.

@Hotell What are examples of this tooling?

ecraig12345 commented:
I just remembered that the pipelines UI supports trigger overrides, so it's actually possible to use the one `azure-pipelines.yml` against multiple pipelines with different triggers. 

Now I've reverted to the single file and updated it to only include a trigger for PRs, and changed the [new CI pipeline](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=164&_a=summary) to use the same file but with a [trigger override in the UI](https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers).

ling1726 commented:
> I just remembered that the pipelines UI supports trigger overrides, so it's actually possible to use the one `azure-pipelines.yml` against multiple pipelines with different triggers.
> 
> Now I've reverted to the single file and updated it to only include a trigger for PRs, and changed the [new CI pipeline](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=164&_a=summary) to use the same file but with a [trigger override in the UI](https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers).

Does that mean we'll have `Fluent UI PR` and `Fluent UI CI` pipelines ?

layershifter commented on azure-pipelines.yml:
```diff
@@ -1,11 +1,17 @@
+trigger: none
 
 variables:
+  - ${{ if ne(variables['Build.SourceBranch'], 'refs/heads/master') }}:
```
Nice 👍 Closed #17011.

ecraig12345 commented:
> > I just remembered that the pipelines UI supports trigger overrides, so it's actually possible to use the one `azure-pipelines.yml` against multiple pipelines with different triggers.
> > Now I've reverted to the single file and updated it to only include a trigger for PRs, and changed the [new CI pipeline](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=164&_a=summary) to use the same file but with a [trigger override in the UI](https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers).
> 
> Does that mean we'll have `Fluent UI PR` and `Fluent UI CI` pipelines ?

Yes

ling1726 commented:
> > > I just remembered that the pipelines UI supports trigger overrides, so it's actually possible to use the one `azure-pipelines.yml` against multiple pipelines with different triggers.
> > > Now I've reverted to the single file and updated it to only include a trigger for PRs, and changed the [new CI pipeline](https://dev.azure.com/uifabric/fabricpublic/_build?definitionId=164&_a=summary) to use the same file but with a [trigger override in the UI](https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers).
> > 
> > 
> > Does that mean we'll have `Fluent UI PR` and `Fluent UI CI` pipelines ?
> 
> Yes

In this case can we make each pipeline supply the `isPr` variable ? would remove the weird branch comparison
